src/celeritas/phys/PhysicsData.hh

```diff
@@ -221,7 +233,7 @@ struct PhysicsParamsScalars
         return max_particle_processes > 0 && model_to_action >= 3
                && num_models > 0 && scaling_min_range > 0
                && scaling_fraction > 0 && energy_fraction > 0
-               && linear_loss_limit > 0
+               && linear_loss_limit > 0 && secondary_stack_factor > 0
```

Comment:
Same here? - also, if it is set to 3 above, should it be checked again here?

Reply:
The `3` is just the default value for the user input; any positive value should be valid.